storybook/stories/Button/index.android.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
Is it needed to make this file for android and ios separately?

Reply:
Probably not, I will remove them